cmd/eval-dev-quality/cmd/evaluate_test.go

```diff
@@ -162,7 +162,10 @@ func TestEvaluateExecute(t *testing.T) {
 				require.NoError(t, err)
 			}
 			sort.Strings(actualResultFiles)
-			expectedResultFiles := make([]string, 0, len(tc.ExpectedResultFiles))
+			var expectedResultFiles []string
```

Comment:
Why is that change necessary?

Reply:
Cause `osutil.FilesRecursive` returns a `nil` list for the actual files but we then call `make` with the length (0) but that is an empty list... not a `nil` list. Hence the assertion would fail.